### PR TITLE
create new prediction_types for mean and median

### PR DESCRIPTION
**Pull request by matthewcornell**
closes [create new prediction_types for mean and median #367]. removes the beta feature to download the Zoltar database for local use

### CODE REVIEW

File: docs/FileFormats.md [hunk 1]
```diff
@@ -71,6 +71,7 @@ The `"predictions"` list contains objects for each prediction, and each object c
     - `"point`: A numeric point prediction with a single `value` key.
```
Comment (nickreich) [inline docs/FileFormats.md:71]: are we missing a close quote here?

Comment (nickreich) [reply docs/FileFormats.md:71]: it should be `"point"` not `"point`

File: docs/FileFormats.md [hunk 2]
```diff
@@ -71,6 +71,7 @@ The `"predictions"` list contains objects for each prediction, and each object c
...
     - `"sample"`: Numeric samples represented as a table with one column that is found in the `sample` key.
     - `"quantile"`: A quantile distribution with two paired columns: `quantile` and `value`.
+    - `"mean"`, `"median"`, and `"mode"`: Similar to `"point"` but with clear semantics.
```
Comment (nickreich) [inline docs/FileFormats.md:74]: I suggest making this line stand on its own, if nothing else to ready ourselves for a removal of the "point" row that might be removed at a later point. So maybe the text after the full colon could read:
"A numeric prediction with a single `value` key, indicating the summary statistic indicated by the name, e.g. the mean."

File: docs/ForecastQueryFormat.md [hunk 3]
```diff
@@ -92,35 +92,31 @@ Some features (currently only automatic prediction type conversion) require addi
 ```
 
 
-### Automatic prediction type conversion
+### Prediction type conversion
```
Comment (nickreich) [inline docs/ForecastQueryFormat.md:95]: Could we add something in this section about how the conversion is requested? It's not clear to me what tool someone would use to activate the conversion of point types. Maybe this should be a separate issue.

File: docs/Validation.md [hunk 4]
```diff
@@ -61,7 +61,7 @@ Bernoulli   | 0<=p<=1   |    -      |    -
 Binomial    | 0<=p<=1   |   n>0     |    - 
 -->
 
-### `Point` Prediction Elements
+### `Point` Prediction Elements (also applies to `mean`, `median`, and `mode`)
```
Comment (nickreich) [inline docs/Validation.md:64]: Same as comment above to feature the new types more on equal footing with point, could we make this header

### `point`, `mean`, `median`, and `mode` Prediction Elements

File: docs/Validation.md [hunk 5]
```diff
@@ -99,7 +99,7 @@ These tests are performed when a forecast is created or updated.
 
 ## Tests for Prediction Elements by Target Type
 
-These tests are performed when a forecast is created or updated. For all target types, only [valid Prediction Types](Targets.md#valid-prediction-types-by-target) are accepted.
+These tests are performed when a forecast is created or updated. For all target types, only [valid Prediction Types](Targets.md#valid-prediction-types-by-target) are accepted. Note: `point` also applies to `mean`, `median`, and `mode`.
```
Comment (nickreich) [inline docs/Validation.md:102]: I suggest making these changes explicitly in the text below, not just once at the top. Will save us work later and be cleaner now.

**Review by nickreich — CHANGES_REQUESTED** [review]
I have a few minor requested changes. My main overarching comment is that I think there are some places where we could save ourselves time later (when removing or deactivating the point prediction type) by just putting mean/median/mode on equal footing in the documentation now.

**Comment by matthewcornell** [comment]
Thanks very much for the comments, @nickreich . I've incorporated them in the new commit.

**Review by nickreich — APPROVED** [review]
Thanks, @matthewcornell . LGTM!

**Pull request merged**